doc/plugin_server_datastore_sql.md

```diff
@@ -11,8 +11,8 @@ The `sql` plugin implements SQL based data storage for the SPIRE server using SQ
 | client_cert_path     | Path to client certificate (MySQL only)                                                                                                                                                                                                                                            |
 | client_key_path      | Path to private key for client certificate (MySQL only)                                                                                                                                                                                                                            |
 | max_open_conns       | The maximum number of open db connections (default: 100)                                                                                                                                                                                                                           |
-| max_idle_conns       | The maximum number of idle connections in the pool (default: 2)                                                                                                                                                                                                                    |
-| conn_max_lifetime    | The maximum amount of time a connection may be reused (default: unlimited)                                                                                                                                                                                                         |
+| max_idle_conns       | The maximum number of idle connections in the pool (default: 100)                                                                                                                                                                                                                    |
+| conn_max_lifetime    | The maximum amount of time a connection may be reused (default: 30s)                                                                                                                                                                                                         |
```

Comment:
We don't need to make any change in the `conn_max_lifetime`. We need to set ConnMaxIdleTime to 30s and document it (we will not expose a setting for that).